packages/core/i18n/README.md

```diff
@@ -165,6 +165,7 @@ const i18n = createI18n<typeof english>(props.locale || 'en-us', english)
   "test": {
     "withParams": "Good {dayPart}, My name is {name}!",
     "withPluralization": "{count} {count, plural, =0 {versions} =1 {version} other {versions}} available"
+    "withDateTimeFormatting": "{value, date, datetime}",
```

Comment:
Looks good, the last thing is: would be great if you can add a short description on what each of those params in {} is: like 
`value`: - actual value of the date to be present
`date`: flag to indicate that value needs to be treated as a date 
`datetime` - ...???

Reply:
> "withDateTimeFormatting": "{value, date, datetime}",

reading this line in en.json I am still not sure what each of those means. can you add a clarification there.  all the examples that use this key, only pass value, so what are 2-nd and 3-d arguments for?

Reply:
I added https://github.com/Kong/public-ui-components/pull/1012/commits/025f0b29688223ea9c9308a65390af683eca14f2 down in the `## Formatting numbers, dates and times` section, lmk if that helps.